docs/_includes/ds6/notice.html

```diff
@@ -81,7 +81,7 @@ <h4 class="page-notice__status" id="confirmation-status">
                         <p>You can check <a href="#">your listing</a> or <a href="#">sell another item</a></p>
                     </div>
                     <button aria-label="Dismiss notice" class="page-notice__close">
-                        <span></span>
+                        <span class='icon icon--close'></span>
```

Comment:
I don't think we should change the API/markup for notice module. Because the notice icon is always an X there is no need to require the developer to specify the icon, we abstract that detail away like we were doing currently.